docs/website/versioned_docs/version-3.0.0/command-reference/odo-create-namespace.md

```diff
@@ -0,0 +1,32 @@
+---
+title: odo create namespace
+sidebar_position: 3
+---
+
+`odo create namespace` lets you create a namespace/project on your cluster.
```

Comment:
Please explain that it will create a Project resource when the resource type exists in the cluster (generally in OpenShift clusters), or a Namespace resource otherwise